src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -138,6 +138,25 @@
   ],
 
   properties: [
+    {
+      class: 'String',
+      name: 'referenceSummary',
+      section: 'requestDetails',
```

Comment:
can you update the section here to be 'approvalRequestInformation' and set the order to be 21 and the gridColumns to be 6

Reply:
Done.